RecoLocalCalo/HGCalRecAlgos/plugins/alpaka/HGCalRecHitConfigurationESProducer.cc

```diff
@@ -0,0 +1,100 @@
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/SourceFactory.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/ESProducer.h"
+#include "FWCore/Framework/interface/ESTransientHandle.h"
+#include "FWCore/Framework/interface/EventSetupRecordIntervalFinder.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/ESGetToken.h"
+#include "DataFormats/Math/interface/libminifloat.h"
+
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/ESGetToken.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/ESProducer.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/ModuleFactory.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/host.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+#include "CondFormats/HGCalObjects/interface/HGCalConfiguration.h"
+#include "CondFormats/HGCalObjects/interface/HGCalMappingModuleIndexer.h"
+#include "CondFormats/DataRecord/interface/HGCalElectronicsMappingRcd.h"
+#include "CondFormats/DataRecord/interface/HGCalModuleConfigurationRcd.h"      // depends on HGCalElectronicsMappingRcd
+#include "CondFormats/HGCalObjects/interface/HGCalCalibrationParameterHost.h"  // for HGCalConfigParamHost
+#include "CondFormats/HGCalObjects/interface/alpaka/HGCalCalibrationParameterDevice.h"
+
+#include <string>
+#include <iostream>  // for std::cout
+#include <iomanip>   // for std::setw
+//#include <fstream> // needed to read json file with std::ifstream
+//#include <nlohmann/json.hpp>
+//using json = nlohmann::json;
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  namespace hgcalrechit {
+
+    class HGCalConfigurationESProducer : public ESProducer {
+    public:
+      HGCalConfigurationESProducer(const edm::ParameterSet& iConfig) : ESProducer(iConfig) {
+        if (iConfig.exists("gain"))
+          gain_ = iConfig.getParameter<int>("gain");
+        auto cc = setWhatProduced(this);  //HGCalConfigurationESProducer::produce
+        //findingRecord<HGCalModuleConfigurationRcd>();
```

Comment:
Could you remove this commented out line (to avoid any potential future confusion)?